packages/flutter/lib/src/material/toggle_buttons.dart

```diff
@@ -702,6 +712,21 @@ class ToggleButtons extends StatelessWidget {
           children: buttons,
         ),
       );
+
+    final MaterialTapTargetSize resolvedTapTargetSize = tapTargetSize ?? theme.materialTapTargetSize;
+    switch (resolvedTapTargetSize) {
+      case MaterialTapTargetSize.padded:
+        return ConstrainedBox(
+          constraints: const BoxConstraints(),
```

Comment:
This was necessary for some reason. @HansMuller thoughts?

Reply:
I'm not sure. The implication is that the _InputPadding class is not respecting its incoming constraints, so adding this class forces that to happen.